services/src/Microsoft.Azure.IIoT.Services.Processor.Onboarding/src/Properties/launchSettings.json

```diff
@@ -1,27 +1,7 @@
 {
-  "iisSettings": {
```

Comment:
@marcschier, is the change to `launchSettings.json` of onboarding service intentional ? Just asking because I do not see other changes to onboarding service.

Reply:
Good question.  This was an opportunistic change. VS generates one when there is none checked in and for quite some time it appears it wasnt.   This meant at some point in the past a generated one was accidently checked in.  I now updated the one to be the same as the rest of the launchsettings.json in the tree, particularly for non-web services.